actioncable/lib/rails/generators/js/channel_generator.rb

```diff
@@ -0,0 +1,7 @@
+module Js
+  module Generators
+    class ChannelGenerator < ::Rails::Generators::NamedBase
```

Comment:
Yeah, I think that lookup is why, based on my reading of: https://github.com/rails/coffee-rails/pull/72/files#r62856943